index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8"/>
+	<title>Surfing</title>
+	<link rel="stylesheet" type="text/css" href="normalize.css">
+	<link rel="stylesheet" type="text/css" href="index.css">
+	<meta name="viewport" content="width=device-width, initial-scale=1"/>
+</head>
```

Comment:
Please check out the [recommended minimum](https://github.com/joshbuchea/HEAD#recommended-minimum) for the tags present in the `<head>` section.